src/modes/top10/Top10.svelte

```diff
@@ -287,6 +286,41 @@
     font-size: 1.2em;
   }
 
+  table .rank {
+    width: 1em;
+    text-align: right;
+  }
+
+  table .name {
+    width: 10em;
+    text-align: left;
+    white-space: nowrap;
+  }
+
+  table .table-pop-column {
+    width: 7em;
+    text-align: right;
+  }
+
+  table th {
+    text-align: right;
+    white-space: nowrap;
+  }
+
+  table .main-indicator {
```

Comment:
why is the `table` selector in front of the classname selector needed?

Reply:
Generic class names like 'rank' and 'name' needed more qualification.  It's not needed for table-pop-column, but that could just be called 'population'.  (I thought 'pop' might be related to popper at first :))

I was also confused that .table is not used on a table but a div that contains the table.

I tend to make selectors more specific than needed, having experienced accidental collisions in the past, and adding more qualifiers also helps to make the meaning clear to readers. 

It is generally better to avoid use of tag names, e.g. table and th, so they might be replaced, but that's not so likely given table layout.

But I am open to your suggestions.

Reply:
re collisions: Svelte is automatically scoping each CSS class / selector. Thus, I'm usually not worried about CSS collisions.